services/workflows-service/src/providers/file/file.service.ts

```diff
@@ -219,6 +222,16 @@ export class FileService {
       };
     }
 
+    if (
+      provider == 'base64' &&
+      z.string().refine(Base64.isValid).parse(extractBase64Payload(uri))
```

Comment:
Did you mean to use `z.custom` or similar?

Reply:
P2

Reply:
TBD